### PR TITLE
namespace creation UI feedback for validation

### PR DESCRIPTION
**Pull request by mathis-marcotte**
closes https://github.com/StatCan/daaas/issues/769

### CODE REVIEW

**Review by vexingly — APPROVED** [review]
LGTM!

File: components/centraldashboard/public/assets/i18n/languages.json [hunk 1]
```diff
@@ -165,6 +166,7 @@
             "registrationPage.linkDocumentation": "Documentation",
             "registrationPage.linkPrivacy": "Confidentialité",
             "registrationPage.errValidation": "Le nom doit commencer et terminer par des caractères alphanumériques, les tirets ne sont autorisés qu'à l'intérieur. (longueur minimale >= 1)",
+            "registrationPage.errDuplicate": "L'espace de noms \"{namespace}\" existe déjà",
```
Comment (chuckbelisle) [inline components/centraldashboard/public/assets/i18n/languages.json:169]: déjà

**Review by chuckbelisle — CHANGES_REQUESTED** [review]
juste un petite faute dans le string `registrationPage.errDuplicate`

**Review by chuckbelisle — APPROVED** [review]
In dark mode, the "é" was not showing for the word.

**Pull request merged**